apps/vanilla/src/global/components.css

```
@@ -1,28 +1,111 @@
-a,
-:link,
-.link {
-  --link-color: var(--primary-color);
+/* Header and footer styles */
 
-  color: var(--link-color);
-  text-decoration: none;
-  border-bottom: 1px solid var(--link-color);
+.books-header {
+  padding-block: var(--space-5);
+  border-bottom: 1px solid var(--color-border);
 }
 
-a:hover,
-:link:hover,
-.link:hover {
-  --link-color: var(--primary-light-color);
+.books-footer {
+  display: flex;
+  justify-content: space-between;
+  margin-top: var(--space-6);
+  padding: var(--space-6) 0;
+  height: var(--footer-height);
+  border-top: 1px solid var(--color-border);
+}
+
+.footer-link {
+  color: var(--color-on-surface);
+}
+
+.books-footer__info {
+  display: flex;
+  gap: var(--space-8);
 }
 
-.links {
+.books-footer__additional {
   display: flex;
-  flex-wrap: wrap;
-  gap: 16px;
+  place-items: center;
+  gap: var(--space-2);
+}
+
+.books-footer__button {
+  border: none;
+  cursor: pointer;
+  overflow: visible;
+  background: transparent;
+}
+
+/* Book card component */
+
+.book-card {
+  display: flex;
+  flex-direction: column;
+  gap: var(--space-2);
+}
+
+.book-card__img {
+  border-radius: var(--border-radius-md);
+  object-fit: cover;
+  min-height: 250px;
+}
+
+/* Utility classes */
+
+/* Text utlity */
+
+.text-subtitle {
+  font-size: var(--font-size-sm);
+  font-weight: var(--font-weight-semibold);
+  line-height: var(--line-height-lg);
+  text-transform: uppercase;
+}
+
+.text-body {
+  font-size: var(--font-size-md);
+  font-weight: var(--font-weight-regular);
+  line-height: var(--line-height-xl);
+  text-transform: none;
+}
+
+.text-body-sm {
+  font-size: var(--font-size-xs);
+  font-weight: var(--font-weight-medium);
+  line-height: var(--line-height-lg);
+  text-transform: none;
```

Try not to add css properties with default value
```suggestion
font-size: var(--font-size-xs);
  font-weight: var(--font-weight-medium);
  line-height: var(--line-height-lg);
```